beetsplug/subsonicupdate.py

```diff
@@ -25,16 +25,59 @@
 """
 from __future__ import division, absolute_import, print_function
 
-from beets.plugins import BeetsPlugin
-from beets import config
-import requests
-import string
 import hashlib
 import random
+import string
+
+import requests
+
+from beets import config
+from beets.plugins import BeetsPlugin
 
 __author__ = 'https://github.com/maffo999'
 
 
+def create_token():
+    """ Creates salt and token from given password.
```

Comment:
```suggestion
    """Creates salt and token from given password.
```

Typically, the docstring style doesn't use a space after the opening """.